x-pack/test/security_solution_endpoint_api_int/apis/resolver/children.ts

```diff
@@ -42,25 +42,33 @@ export default function resolverAPIIntegrationTests({ getService }: FtrProviderC
       before(async () => {
         // Construct the following tree:
         // Origin -> infoEvent -> startEvent -> execEvent
-        origin = generator.generateEvent();
+        origin = generator.generateEvent({
+          eventsDataStream: EndpointDocGenerator.createDataStreamFromIndex(processEventsIndex),
+        });
         infoEvent = generator.generateEvent({
           parentEntityID: entityIDSafeVersion(origin),
           ancestry: createAncestryArray([origin]),
           eventType: ['info'],
+          eventsDataStream: EndpointDocGenerator.createDataStreamFromIndex(processEventsIndex),
         });
 
         startEvent = generator.generateEvent({
           parentEntityID: entityIDSafeVersion(infoEvent),
           ancestry: createAncestryArray([infoEvent, origin]),
           eventType: ['start'],
+          eventsDataStream: EndpointDocGenerator.createDataStreamFromIndex(processEventsIndex),
         });
 
         execEvent = generator.generateEvent({
           parentEntityID: entityIDSafeVersion(startEvent),
           ancestry: createAncestryArray([startEvent, infoEvent]),
           eventType: ['change'],
+          eventsDataStream: EndpointDocGenerator.createDataStreamFromIndex(processEventsIndex),
         });
-        genData = await resolver.insertEvents([origin, infoEvent, startEvent, execEvent]);
+        genData = await resolver.insertEvents(
+          [origin, infoEvent, startEvent, execEvent],
+          processEventsIndex
```

Comment:
The process index is the default for `insertEvents` but this just makes it more explicit.